driver.sh

```diff
@@ -207,28 +208,28 @@ check_dependencies() {
   }
 
   if [[ -z "${AR:-}" ]]; then
-    for AR in $(gen_bin_list llvm-ar) llvm-ar "${CROSS_COMPILE:-}"ar; do
-      command -v ${AR} 2>/dev/null && break
+    for AR in llvm-ar "${CROSS_COMPILE:-}"ar; do
+      command -v "${AR}" 2>/dev/null && break
     done
   fi
   check_ar_version
-  ${AR} --version
+  "${AR}" --version
 
   if [[ -z "${NM:-}" ]]; then
-    for NM in $(gen_bin_list llvm-nm) llvm-nm "${CROSS_COMPILE:-}"nm; do
-      command -v ${NM} 2>/dev/null && break
+    for NM in llvm-nm "${CROSS_COMPILE:-}"nm; do
+      command -v "${NM}" 2>/dev/null && break
     done
   fi
 
   if [[ -z "${OBJDUMP:-}" ]]; then
-    for OBJDUMP in $(gen_bin_list llvm-objdump) llvm-objdump "${CROSS_COMPILE:-}"objdump; do
-      command -v ${OBJDUMP} 2>/dev/null && break
+    for OBJDUMP in llvm-objdump "${CROSS_COMPILE:-}"objdump; do
+      command -v "${OBJDUMP}" 2>/dev/null && break
     done
   fi
 
   if [[ -z "${OBJSIZE:-}" ]]; then
-    for OBJSIZE in $(gen_bin_list llvm-size) llvm-size "${CROSS_COMPILE:-}"size; do
-      command -v ${OBJSIZE} 2>/dev/null && break
+    for OBJSIZE in llvm-size "${CROSS_COMPILE:-}"size; do
+      command -v "${OBJSIZE}" 2>/dev/null && break
     done
   fi
```

Comment:
do you think that you could write a function that accepted `OBJSIZE` or `OBJDUMP` or ... and did this, in order to DRY up this code?

Reply:
I'll see if I can think of something but

```
VAR=OBJDUMP
echo ${${VAR}}
```

to try and get `${OBJCOPY}` does not work :(

Reply:
The names of the binaries are also different. Ok maybe not as easy to DRY up as I thought.